### PR TITLE
Evernote imports

### PR DESCRIPTION
**Pull request by andrewconner**
@eishay @stkem 

Not promoting it until we QA a bit more (with more than just my notes)

### CODE REVIEW

**Comment by eishay** [comment]
yey

File: kifi-backend/modules/common/app/com/keepit/model/Keep.scala [hunk 1]
```diff
@@ -292,13 +294,11 @@ object KeepSource {
...
 
-  val valid = Set(keeper, bookmarkImport, site, mobile, email, default, bookmarkFileImport, kippt, pocket, instapaper, emailReco, twitterFileImport)
-
-  val imports = Set(bookmarkImport, kippt, pocket, instapaper, bookmarkFileImport, twitterFileImport)
+  val imports = Set(bookmarkImport, kippt, pocket, instapaper, evernote, diigo, bookmarkFileImport, twitterFileImport)
 
   // Sources that are from users uploading files, bulk actions, inputting URLs, etc.
   // These may be old links
-  val bulk = Set(site, bookmarkImport, kippt, pocket, instapaper, bookmarkFileImport, twitterFileImport, userCopied, unknown)
+  val bulk = imports ++ Set(userCopied, unknown, evernote)
```
Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala]: Unless I'm seriously misreading something, `imports` already contains `evernote`. Why add it again?

**Comment by stkem** [comment]
lgtm (no idea what evernotes data format actually looks like though, so I'm taking that part on good faith :-)

File: kifi-backend/modules/common/app/com/keepit/model/Keep.scala (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala]: Mistake :)

**Pull request merged**